src/RetireNet.Packages.Tool/RetireNet.Packages.Tool.csproj

```diff
@@ -26,12 +28,20 @@
         <PackageReference Include="Microsoft.Extensions.Options" Version="2.1.0" />
         <PackageReference Include="Newtonsoft.Json" Version="11.0.2" />
         <PackageReference Include="NuGet.ProjectModel" Version="4.8.0" />
+        <PackageReference Include="Scrutor" Version="3.2.1" />
     </ItemGroup>
 
     <ItemGroup>
         <Content Include="appsettings.json">
             <CopyToOutputDirectory>Always</CopyToOutputDirectory>
         </Content>
     </ItemGroup>
+    
+    <ItemGroup>
+        <None Include="images\icon.512x512.png" Visible="false">
+            <PackagePath />
+            <Pack>true</Pack>
+        </None>
+    </ItemGroup>
```

Comment:
How come you added this?

Reply:
I was trying to get rid of the notice in the build about the Icon tag being deprecated. It can be tossed, I have no emotional attachment to it, and it didn't shut up the warning, I just forgot about it.